### DISCUSSION TITLE
If my screen resolution is 5760x1080, the "merge stacks" button not working anymore

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13824] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

1. The merge stacks button stopped working

### Reproduction steps
1. use screen resolution is 5760x1080
2. divide any stack  of items in a shelf in two and try to press "merge stacks" button.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
I can't use 5760x1080 on normal monitor. It's too distorted to even open a cabinet
Do you mean button cannot be pressed?
![20240425090418_1](https://github.com/FakeFishGames/Barotrauma/assets/122838333/b787fddb-c56f-4971-8e47-08dd3868915b)

**Reply by pavel4win** [reply]
yes the "sort" button can be pressed but the button "merge" can not be pressed
5760x1080 not only resolution with this error.
I think this happens in some non standard resolutions
As example 3072x768 also has this error. I think this resolution can be used on any 4k monitors.
In 3072x768 button "merge" still works but "active field" shifted up relative to the icon.
So, this is a scaling error.
I think in 5760x1080  res "merge" button's "active field" shifted up too high under the window border,
although the "sort" button works fine

My screen resolution is 5760x1080 it is 3 monitors 1920x1080 connected to one AMD video card and merged with AMD eyefinity

**Comment by pavel4win** [comment]
In 5760x1080 res there are several scaling issues.
Now I'll give you some screenshots.

1. When you take metal crate it's window prevents interaction with the environment. 
First you have to move the window to the left

![2024-04-27_20-33-04](https://github.com/FakeFishGames/Barotrauma/assets/47295383/2bf970ee-1547-4611-a6ca-b6b6681683f8)

2. When use medical interface it is easy to wear off dress, helmet, headset or ID card because 
the "medicine use field" is superimposed on the inventory.
![2024-04-27_20-35-40](https://github.com/FakeFishGames/Barotrauma/assets/47295383/aae903fc-5a80-4839-abfe-b80a69804a84)

This also happens when treating a friend.
![2024-04-27_20-40-23](https://github.com/FakeFishGames/Barotrauma/assets/47295383/452c1636-b72f-4d00-b6ff-8ff6fe063895)

However, all these problems are not so terrible.
The enemies can be seen from afar if they are in front and behind.
The “presence effect” is unbelievable.

**Comment by pavel4win** [comment]
The merge button still doesn't work in the resolution 5760x1080

**Comment by pavel4win** [comment]
After last update 1.6.17 merge button also not work in the resolution 5760x1080

**Reply by SomeRandomNoobKekeke** [reply]
> I can't use 5760x1080 on normal monitor. It's too distorted to even open a cabinet

Devs probably also can't

No one has 5760x1080 but you, so fix it yourself
![BaroDev (wide)](https://github.com/user-attachments/assets/8a8c1b2e-cdca-4a6f-ac66-e19dbfd989d2)

**Reply by Regalis11** [reply]
Yeah, unfortunately we don't have access to monitors that support these kinds of resolutions, so that makes fixing the issues quite difficult. :/ It's also not a very high-prio issue to address, considering these sorts of resolutions are so rare. I would personally recommend using a more "normal" resolution until we get around to addressing these problems. Sorry for the inconvenience!

**Reply by pavel4win** [reply]
As example 3072x768 resolution also has this error. I think this resolution can be used on any 4k monitors.
There is the problem with scaling. With determining the position of the cursor in the window or something like that.
At some low resolutions, the merge button problem also appears. It works but is shifted relative to the displayed icon.
I agree that the priority is not very high.
I just reminded you of my problem.

I connected three 1920*1080 monitors into one using the video card driver. My card is AMD Radeon RX580.
I really enjoy playing at this resolution and am willing to put up with any scaling issues.
Any modern video card with three outputs allows this.
You can do it if you try.
